docker-startup.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+i=1
+MAXCOUNT=60
+echo "Waiting for PostgreSQL to be running"
+while [ $i -le $MAXCOUNT ]
+do
+  pg_isready -q && echo "PostgreSQL running" && break
+  sleep 2
+  i=$((i+1))
+done
+test $i -gt $MAXCOUNT && echo "Timeout while waiting for PostgreSQL to be running"
+
+case "$1" in
+import)
+  psql -c "SELECT 1 FROM pg_database WHERE datname = 'planet';" | grep -q 1 && dropdb planet
+
+  python3 makedb.py -j 4 -f data.osm.pbf db import
+  python3 makedb.py db prepare
+
+  cd  /tmp
+  curl -s http://www.nominatim.org/data/country_grid.sql.gz > country_grid.sql.gz && \
+    gunzip country_grid.sql.gz
+
+  psql -d planet -c "DROP TABLE IF EXISTS country_osm_grid" && \
+    psql -d planet -f country_grid.sql && \
```

Comment:
If you do `zcat /home/suzuki/osm/dev/Nominatim/data/country_osm_grid.sql.gz | psql -d planet` you don't need to keep the unzipped file around.